developer/frameworks/articles/09-info-framework/01-the-info-framework-intro.markdown

```diff
@@ -0,0 +1,81 @@
+# The Info Framework
+
+The Info Framework provides enhances flexibility for retrieving, processing, 
+and displaying any information, regardless of where the information comes from. 
+The Info Framework can process data from the database, created 
+through process in memory, or retrieved from an external source.
+
+The Info Framework is a new feature @product-ver@ and currently has limited 
+functionality. The current version is the first set for obtaining and 
+displaying information from external systems or custom data models. The Info 
+framework is also very lightweight, and  does not require you to implement any  
+specific interface to use it. That means you can use it for an object even if 
+you don't have access to the source. 
+
+Rather than create a monolithic framework which requires all of the features to 
+be implemented for the sake of using any, it is a collection of loosely coupled 
+micro-frameworks. This lowers the learning curve and minimizes work for 
+developers.
+
+| Note: Liferay veterans may notice some similarities between the [Asset Framework](asset-framework-link) and Info 
+| Frameworks. The Info Framework doesn't yet provide the full array of features 
+| that Assets provide, but it has many similar characteristics and is simpler 
+| to use.
+
+## Using the Info Framework
+
+The Info Framework simplifies information handling across the board. You can 
+use it to customize your own applications, and it provides new ways to use 
+existing Liferay features. In previous versions of Liferay, features like 
+Display Pages and the Asset Publisher could only be used if you create an 
```

Comment:
And that's still true for Asset Publisher. It's only been generalized for Display Pages.

Reply:
You had spelled out a couple different ways that the Info Framework can interact with the Asset Publisher, so, just to be clear on this one, the Info Framework can still interact with the Asset Publisher but only when used in conjunction with an existing Asset?

Reply:
Instead of talking about the Info Framework in general, we are talking here about the InfoListProvider. The asset publisher can leverage instances of InfoListProviders as long as they return lists of AssetEntry. It will ignore all other providers.